packages/phoenix-evals/src/phoenix/evals/models/mistralai.py

```diff
@@ -73,19 +69,14 @@ def _model_name(self) -> str:
 
     def _init_client(self) -> None:
         try:
-            from mistralai.async_client import MistralAsyncClient
-            from mistralai.client import MistralClient
-            from mistralai.exceptions import MistralAPIException
-            from mistralai.models.chat_completion import ChatMessage
+            from mistralai import Mistral
         except ImportError:
             self._raise_import_error(
                 package_name="mistralai",
                 package_min_version=MINIMUM_MISTRAL_VERSION,
             )
-        self._client = MistralClient()
-        self._async_client = MistralAsyncClient()
-        self._ChatMessage = ChatMessage
-        self._MistralAPIException = MistralAPIException
+
+        self._client = Mistral(os.getenv("MISTRAL_API_KEY"))
```

Comment:
does this not happen automatically anymore? should we be able to pass the key in explicitly?

Reply:
It looks like it doesn't based on my testing. The Mistral docs show storing the key in environment variables, but then [all the examples](https://github.com/mistralai/client-python/tree/main?tab=readme-ov-file#sdk-example-usage) manually load the key from environment variables

Reply:
we might want to allow users to pass the key manually IMO, in case that's what they prefer?